### ISSUE TITLE
Power toggle button in realtime effect dialogs does not have an access key

### ISSUE THREAD

**Issue by DavidBailes** [audacity/audacity#3491]
### Steps to reproduce

Open a realtime effects dialog

### Expected behavior

Power toggle button has an access key

### Actual behavior

Power toggle button does not have an access key

### Audacity Version

current master / alpha / nightly build

### Operating system

Windows

### Additional context

The power toggle button replaces the enable check box in previous versions of Audacity.
The enable check box had the access key E.
If an access key other than E is chosen for the power toggle button, then there will be the need to check all the built in effects which may become realtime to make sure access keys remain unique in a dialog.

For buttons which don't have a text label, then wxWidgets does not display any access key visually, but it can be read by screen readers.

See also:
https://github.com/audacity/audacity/issues/3482
https://github.com/audacity/audacity/issues/3492

**Comment by vsverchinsky** [comment]
@LWinterberg 
'E' key now is reserved for hiding and showing effects panel. Command shortcuts are global and I made them available in effect settings dialog here https://github.com/audacity/audacity/issues/3483.

**Comment by LWinterberg** [comment]
I think e would still work; note that this is an access key and not a shortcut. Access keys are used by holding Alt and the key, and Alt+E is free

**Comment by LWinterberg** [comment]
Hello! We just [switched development to Audacity 4](https://forum.audacityteam.org/t/master-has-been-switched-to-4-0-0-development/120444). We've redone, or plan on redoing, various things from scratch, including the entire UI, and the part this bug is about. As such, I'm closing this bug as it will be fixed there and we won't be fixing it for the final releases of Audacity 3.
